src/solver/hydro/daily/h2o_j_initialiser_les_bornes_des_variables.cpp

```diff
@@ -53,7 +53,7 @@ void H2O_J_InitialiserLesBornesdesVariables(DONNEES_MENSUELLES* DonneesMensuelle
         int Var = CorrespondanceDesVariables.NumeroDeVariableTurbine[Pdt];
         Xmax[Var] = TurbineMax[Pdt];
 
-        Xmin[Var] = std::min(TurbineMax[Pdt], std::max(TurbineCible[Pdt], TurbineMin[Pdt]));
+        Xmin[Var] = std::min(TurbineMax[Pdt], TurbineMin[Pdt]);
```

Comment:
I agree, since there is a precheck for each hour in `checkGenerationPowerConsistency` function, I don't see how TurbineMax in any situation can be lower that TurbineMin, but maybe I am wrong.